leaf-ui/analysis.html

```diff
@@ -57,7 +57,8 @@
                 <!-- slider div -->
                 <input type="range" min="0" max="2" value="0" class="EVO-slider" id="colorResetAnalysis">
                 <!-- makes slider -->
-                <p id="top-label-analysis" style="display: inside">EVO</p> <!-- sat slider label -->
+                <p id="top-label-analysis" style="display: inside">EVO</p>
+                <!-- sat slider label -->
```

Comment:
Does this label apply to line 60 or 62? Why move it

Reply:
line 60, just to make it the same format as other docstrings